### PR TITLE
option and settings

### PR DESCRIPTION
**Pull request by IzaacAyelin**
***Ignore PR title**

Second Media implementation:

- GalleryItem -  creating a gallery item out of the second media dto and make sure it is updated when + make some 
adjustment to the renderProps function
- Create a wrapping component that will take text/image item and will render it with and overlapping second image/text item
- tests and mocks

### CODE REVIEW

File: packages/layouts/src/classes/galleryItem.js [hunk 1]
```diff
@@ -106,21 +147,23 @@ class GalleryItem {
...
       isDemo: this.isDemo,
       videoUrl: this.videoUrl,
       isExternalVideo: this.isExternalVideo,
+      hasSecondaryMedia: this.hasSecondaryMedia,
       ...config,
     };
+    if (this.hasSecondaryMedia) {
+      itemProps.secondaryMediaItem = {
+        ...this.secondaryMediaItem.renderProps(),
+        style: {
```
Comment (IzaacAyelin) [inline packages/layouts/src/classes/galleryItem.js:160]: should be specific

Comment (IzaacAyelin) [reply packages/layouts/src/classes/galleryItem.js:160]: Spreading the secondaryMedia props and then adding the style from the main item except the bgColor that has to be the secondarymedia item's bgColor

**Review by nirnaor — APPROVED** [review]

**Comment by nirnaor** [comment]
@IzaacAyelin you can click the "edit button" to change the PR title :)

**Comment by IzaacAyelin** [comment]
> @IzaacAyelin you can click the "edit button" to change the PR title :)

I can't believe I missed that

**Comment by ErmiaJoey** [comment]
Use a wrapping component like OverlappingMedia to handle text or [shoe brand](https://stylo.pk/) image items with overlapping effects.

**Pull request merged**